spikeinterface/core/basesorting.py

```diff
@@ -274,6 +295,8 @@ def to_spike_vector(self, extremum_channel_inds=None):
         
         if extremum_channel_inds is not None:
             spike_dtype += [('channel_ind', 'int64')]
+        if count_spike_nb:
+            spike_dtype += [('spike_nb', 'int64')]
```

Comment:
@Dradeliomecus what is this for? and what does `spike_nb` represent?

Reply:
This means "What's the index of this spike in the unit spike train".
I suck at finding names...

Reply:
by unit?